src/server/controllers/tasks.js

```diff
@@ -52,4 +52,49 @@ export class TaskManager {
       }
     });
   }
+
+
+  /**
+     *
+     *
+     * @param {*} [option={}]
+     * @memberof TaskManager
+     */
+  updateATask() {
+    Task.findById(this.req.params.id).exec((error, tasks) => {
```

Comment:
The property is still named `tasks` yet it'll return only one task.

Reply:
sorry will change that too